LDAR_Sim/src/virtual_world/equipment_groups.py

```diff
@@ -83,15 +83,20 @@ def _update_prop_params(self, info: dict, prop_params: dict) -> None:
 
         prop_params[cp.METH_SPECIFIC] = meth_specific_params
 
-    def _create_equipment(self, infrastructure_inputs, prop_params, info) -> None:
-        tot_equip = info.sum()
+    def _create_components(
+        self, infrastructure_inputs: dict, prop_params: dict, info: pd.Series
+    ) -> None:
+        cleaned_info = self._clean_propagating_parameters_from_equipment_info(info)
+        total_components = cleaned_info.sum()
         nonrep_epr = prop_params[IC.Equipment_Group_File_Constants.NON_REP_EMIS_EPR]
         rep_epr = prop_params[IC.Equipment_Group_File_Constants.REP_EMIS_EPR]
         if nonrep_epr is not None and nonrep_epr > 0:
-            prop_params[IC.Equipment_Group_File_Constants.NON_REP_EMIS_EPR] = nonrep_epr / tot_equip
+            prop_params[IC.Equipment_Group_File_Constants.NON_REP_EMIS_EPR] = (
```

Comment:
Nit: This might be better in a for-loop, similar to how we did the sites. 
ex 
```python
for param in Infrastructure_Constants.Sites_File_Constants.PROPAGATING_PARAMS_TO_SCALE:
                    if prop_params[param] is not None:
                        prop_params[param] /= equipment_groups```

Reply:
Not going to change this as part of this PR but it is something we should do